lib/plugins/inspec-init/templates/plugins/inspec-plugin-template/test/fixtures/README.md

```diff
@@ -17,7 +17,7 @@ In this directory, you would place things that you need during testing.  For exa
 
 When writing your functional tests, you can point InSpec at the various test fixture profiles, and know that when it points at the zero-controls profile, it should find no controls; and when pointed at the twelve-controls profile, it should find 12.
 
-## Using test fixtures provided with core inspec
+## Using test fixtures provided with the `inspec` source code
```

Comment:
I'm confused why this referring to the command and not `with InSpec source code`. I'll leave it be though, not a blocking issue.